src/functional/ffront/decorator.py

```diff
@@ -283,6 +283,11 @@ def _validate_args(self, *args, **kwargs) -> None:
     def _process_args(self, args: tuple, kwargs: dict) -> tuple[tuple, tuple, dict[str, Any]]:
         self._validate_args(*args, **kwargs)
 
+        initialized_domain = True
+        for _i, body_entry in enumerate(self.past_node.body):
```

Comment:
not relevant, as I propose to remove the for loop anyway, but here it should be just
```suggestion
        for body_entry in self.past_node.body:
```
No need to enumerate.